src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,19 @@
-export const TodoInfo = () => {};
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const todoStatus = todo.completed
+    ? 'TodoInfo TodoInfo--completed'
+    : 'TodoInfo';
```

Comment:
Use classnames to set classes conditionally.
It's not necessary to create a variable, just setting in classnames is enough, but it's up to you.